src/Tribe/Linked_Posts/Chooser_Meta_Box.php

```diff
@@ -236,7 +236,11 @@ protected function move_handle() {
 	 *
 	 */
 	protected function delete_handle() {
-		echo '<a class="dashicons dashicons-trash tribe-delete-this" href="#"><span class="screen-reader-text">Delete This</span></a>';
+	    ?>
+		<a class="dashicons dashicons-trash tribe-delete-this" href="#">
+            <span class="screen-reader-text"><?php esc_html_e( 'Delete this', 'the-events-calendar' ); ?></span>
+        </a>
```

Comment:
On this line and the next, the initial indent uses only spaces (but should use tabs)